### PR TITLE
Add configs for ond18

### PR DESCRIPTION
**Pull request by as6520**
This PR adds configs for 18 month eval for image classification. Depends on https://github.com/darpa-sail-on/evm_based_novelty_detector/pull/147

### CODE REVIEW

File: configs/problem/ic_ond18_system_detection.yaml [hunk 1]
```diff
@@ -23,15 +23,15 @@ detectors:
...
          cover_threshold: 0.8
-         distance_multiplier: 0.65
+         distance_multiplier: 0.6
          distance_function: "cosine"
          chunk_size: 200
          gpu_list:
            - 0
            - 1
          mu_train: 1.0
          sigma_train: 0.028588828
```
Comment (owl-uccs) [inline configs/problem/ic_ond18_system_detection.yaml:33]: 0.09128787

File: configs/problem/ic_ond18_given_detection.yaml [hunk 2]
```diff
@@ -25,15 +25,15 @@ detectors:
...
          cover_threshold: 0.8
-         distance_multiplier: 0.65
+         distance_multiplier: 0.6
          distance_function: "cosine"
          chunk_size: 200
          gpu_list:
            - 0
            - 1
          mu_train: 1.0
          sigma_train: 0.028588828
```
Comment (owl-uccs) [inline configs/problem/ic_ond18_given_detection.yaml:35]: 0.09128787

File: configs/problem/ic_ond18_reaction.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      cnn_params:
+          model_path: ""
+          known_classes: 413
+          known_unknown_classes: 51
+          image_size: 320
+      evm_params:
+         model_path: ""
+         tail_size: 40000
+         cover_threshold: 0.8
+         distance_multiplier: 0.65
```
Comment (owl-uccs) [inline configs/problem/ic_ond18_reaction.yaml:39]: 0.6

File: configs/problem/ic_ond18_reaction.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+         distance_function: "cosine"
+         chunk_size: 200
+         gpu_list:
```
Comment (owl-uccs) [inline configs/problem/ic_ond18_reaction.yaml:42]: multi gpu option removed. now it work on only in single gpu

File: configs/problem/ic_ond18_reaction.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+           - 0
+           - 1
+         mu_train: 1.0
+         sigma_train: 0.028588828
```
Comment (owl-uccs) [inline configs/problem/ic_ond18_reaction.yaml:46]: this number and following are old

Comment (as6520) [reply configs/problem/ic_ond18_reaction.yaml]: Addressed in a8d4a29

File: configs/problem/ic_ond18_reaction.yaml (hunk 4 above)
Comment (as6520) [reply configs/problem/ic_ond18_reaction.yaml]: Addressed in a8d4a29

File: configs/problem/ic_ond18_reaction.yaml (hunk 3 above)
Comment (as6520) [reply configs/problem/ic_ond18_reaction.yaml]: Addressed in a8d4a29

**Review by owl-uccs — APPROVED** [review]

**Pull request merged**